makefile

```diff
@@ -57,29 +57,41 @@ binn.o: src/binn.c src/binn.h
 dllmain.o: src/win32/dllmain.c
 	$(CC) -Wall -c $<
 
-install:
+install: binn.pc-install
```

Comment:
this will make the installation depend on `sed`. is it common?

Reply:
yes, sed is widely available, works with the busybox sed implementation as well

I did not test any of the changes with windows tho...

Reply:
no problem, install on windows is not supported